index.html

```diff
@@ -30,10 +47,26 @@
           }
           const r = res.get(key);
           r.data.push(value);
-          r.revision.push(entry.revision.substr(0, 6));
+          r.revision.push(entry.revision.substr(0, 7));
         }
       }
-      return res
+      return res;
+    }
+
+    function show_clipboard_text(text) {
+      var notifyClipboardTextElem = document.getElementById("notify-clipboard-text");
+      notifyClipboardTextElem.textContent = text + " copied to clipboard";
```

Comment:
```suggestion
```

Reply:
Message was given from higher up so that `show_notification` function is reusable. Notification text was formed using template strings.